components/__tests__/RadioButtonGroup-test.js

```diff
@@ -26,30 +26,30 @@ describe('RadioButtonGroup', () => {
 
       it('sets disabled attribute if disabled prop is set', () => {
         wrapper.setProps({ disabled: true });
-        expect(div.props().disabled).toEqual(true);
+        expect(wrapper.first('dev').props().disabled).toEqual(true);
       });
     });
 
     describe('children', () => {
-      const radioButton = wrapper.find(RadioButton);
+      const radioButton = () => wrapper.find(RadioButton);
 
       it('renders expected number of children', () => {
-        expect(radioButton.length).toEqual(2);
+        expect(radioButton().length).toEqual(2);
       });
 
       it('should set checked property based on defaultSelected prop', () => {
-        expect(radioButton.last().props().checked).toEqual(true);
+        expect(radioButton().last().props().checked).toEqual(true);
       });
 
       it('should set checked property based on valueSelected prop', () => {
         wrapper.setProps({ valueSelected: 'male' });
```

Comment:
Not really related to this PR, but could we use a different example? Enforcing binary gender selection is problematic, and while I realize it's just for testing, it's a pattern we should get out of the habit of using.

Reply:
Agreed! Will take a todo to update tests like this.